lib/optimizely.rb

```diff
@@ -131,6 +131,11 @@ def get_variation(experiment_key, user_id, attributes = nil)
         return nil
       end
 
+      unless user_id.is_a? String
```

Comment:
@mikeng13 Fixing 'present' method, showed up some other apparently invalid scenarios, where a user id assigned a variation even when its null/boolean. The documentation says it has to be a string. Let me know if this is the intended behavior.